lib/__tests__/standalone.test.js

```diff
@@ -229,28 +229,27 @@ it("unknown formatter option", () => {
     });
 });
 
-it("standalone with different configs per file", () => {
+describe("standalone with different configs per file", () => {
   let results;
 
   beforeEach(() => {
     return standalone({
-      code: "a {}",
-      config: configBlockNoEmpty
+      files: `${fixturesPath}/config-per-file/**/*.css`
```

Comment:
For reference, [this configuration](https://github.com/stylelint/stylelint/commit/d8124772d98b99b9dcadc58f0980c4265769752d#diff-5b0cfd9051e8587f023001014976e639L211) was [never migrated](https://github.com/stylelint/stylelint/commit/d8124772d98b99b9dcadc58f0980c4265769752d#diff-ef9d81d50b3421b5dfd60c769c7c5a50R234) over when stylelint switched to Jest for the tests in #2083. 

This test here for all intents and purposes has not been functioning as originally intented with the fixtures in the fixture tree at https://github.com/stylelint/stylelint/tree/master/lib/__tests__/fixtures/config-per-file/a since